gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -211,6 +211,13 @@ typedef struct _GstTensorFilterFramework
        * @param[in/out] private_data A subplugin may save its internal private data here. The subplugin is responsible for alloc/free of this pointer. Normally, close() frees private_data and set NULL.
        * @return 0 if ok. < 0 if error.
        */
+
+  int (*checkHWSupport) (accl_hw hw);
+      /**< Mandatory. Check if the provided hardware accelerator is supported. This check can be static or dyanmic based on framework support. Positive response of this check does not guarantee successful running of model with this accelerator. This check can be performed without opening the framework.
```

Comment:
typo dyanmic

Reply:
I think it's `Optional`, not `Mandatory`.

Reply:
Updated.